ansible/deploy-orchestration.yml

```diff
@@ -0,0 +1,33 @@
+---
+- include: ansible-version.yml
+
+- hosts: orchestration
+  gather_facts: true
+  roles:
+    - docker_py
```

Comment:
I think I was previously using this group as a "hack" to run dom0 and gather_facts pre-emptively. I guess it can be removed.